bin/src/data/csv.py

```diff
@@ -198,10 +196,10 @@ def parse_csv_to_input_label_meta(self, load_method: Any) -> Tuple[dict, dict, d
             elif category.lower() == "label":
                 label_data[f"{name}:{data_type}"] = data[key]
             elif category.lower() == "meta":
-                meta_data[f"{name}:{data_type}"] = data[key]
+                meta_data[f"{name}"] = data[key]
```

Comment:
why not data type on meta?, maybe just minor comment line above

Reply:
The type of meta should not matter for the training, however, we have to think of a way to make it work for downstream analysis.

Reply:
Since this is the csvloader, it should be general, so meta should still keep data type